README.md

```diff
@@ -172,6 +172,19 @@ OPTIONS:
 EXAMPLE: run.sh -s "Squeak-trunk" --headfull /path/to/projects
 ```
 
+## Running locally
+
+If you need to run smalltalkCI locally you can follow these steps:
+
+1. clone your repository
+  * a) from github: `git clone https://github.com/your/project.git`, or
+  * b) clone your local repo: `git clone /wherever/is/your/project`, or
+  * c) use your local repo directly
+2. cd into the cloned directory `cd project`
+3. download smalltalkCI: `wget -O smalltalkCI.zip https://github.com/hpi-swa/smalltalkCI/archive/master.zip`
+4. unzip it: `unzip -o smalltalkCI.zip`
+5. run smalltalkCI, provide dialect you want to test and your local config: `smalltalkCI-master/run.sh --headfull -s "Pharo-5.0" .smalltalk.ston`
+
```

Comment:
Let's not overthink this :)
It's really up to the user if he wants to download or git clone smalltalkCI, so why not mention both?
Once on a local filesystem, the user can initiate a build via `/path/to/smalltalkci/run.sh -s {SMALLTALK_VERSION} /path/to/my/projects/smalltalk.ston` (needs double-checking; I'm on a road trip, so I'm writing this from the top of my head).
Running Pharo and Squeak builds should be safe, I wouldn't recommend running GemStone except if you know what you are doing ;) (GemStone has more OS requirements).